docs/src/main/asciidoc/features.adoc

```diff
@@ -8,18 +8,30 @@
 2016-02-02 15:31:01.936  INFO [bar,46ab0d418373cbc9,46ab0d418373cbc9,false] 23030 --- [nio-8081-exec-4] ...
 ----
 +
-(notice the `[appname,traceId,spanId,exportable]` entries from the MDC).
+(notice the `[appname,traceId,spanId,exportable]` entries from the MDC. `appname`, `traceId` , `spanId` seem to be quite
+straightforward, `exportable` means whether the log should be exported to Zipkin or not).
 
-* Optionally log span data in JSON format for harvesting in a log aggregator (set `spring.sleuth.log.json.enabled=true`).
+* Provides an abstraction over common distributed tracing data models: traces, spans (forming a DAG), annotations,
```

Comment:
Not sure if it goes here, but here's a stab..

Sleuth records timing information to aid in latency analysis. Using sleuth, you can pinpoint causes of latency in your applications. Sleuth is written to not log too much, and to not cause your production application to crash.
* propagates structural data about your call-graph in-band, and the rest out-of-band.
* includes opinionated instrumentation of layers such as HTTP 
* includes sampling policy to manage volume
* can report to a Zipkin system for query and visualization